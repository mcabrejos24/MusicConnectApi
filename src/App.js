import React, {useEffect} from 'react'
import { BrowserRouter, Route, Switch } from 'react-router-dom';
import './assets/styles/App.scss';

import Navbar from './components/Navbar';

import Home from './scenes/HomeSPA/Home';
import OnHere from './scenes/HomeSPA/OnHere';
import Email from './scenes/HomeSPA/Email';
import Texting from './scenes/HomeSPA/Texting';

import Donate from './scenes/Donate';
import ContactUs from './scenes/ContactUs';
import AboutUs from'./scenes/AboutUs';
import Redirect from'./scenes/Redirect';


function App() {
  return (
    <BrowserRouter>
      <div className="container">
        <Navbar />
        <Switch>
          <Route path="/" component={ Home } exact={ true } />
          <Route path="/on-here" component={ OnHere } />
          <Route path="/email" component={ Email } />
          <Route path="/texting" component={ Texting } />
          <Route path="/donate" component={ Donate } />
          <Route path="/contact-us" component={ ContactUs } />
          <Route path="/about-us" component={ AboutUs } />
          <Route path="/redirect" component={ Redirect } />
        </Switch>
      </div>
    </BrowserRouter>
  );
}

export default App;
