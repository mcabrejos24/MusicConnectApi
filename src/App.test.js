import App from './App';
import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("tests App.js", () => {
  it("renders App without crashing", () => {
    shallow(<App />);
  });
  it("compares App snapshot to check if it renders correctly", () => {
    const tree = shallow(<App />);
    expect(toJson(tree)).toMatchSnapshot();
  });
})
