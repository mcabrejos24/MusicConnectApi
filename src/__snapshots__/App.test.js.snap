// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests App.js compares App snapshot to check if it renders correctly 1`] = `
<BrowserRouter>
  <div
    className="container"
  >
    <Navbar />
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        path="/on-here"
      />
      <Route
        component={[Function]}
        path="/email"
      />
      <Route
        component={[Function]}
        path="/texting"
      />
      <Route
        component={[Function]}
        path="/donate"
      />
      <Route
        component={[Function]}
        path="/contact-us"
      />
      <Route
        component={[Function]}
        path="/about-us"
      />
      <Route
        component={[Function]}
        path="/redirect"
      />
      <Route
        component={[Function]}
        path="/privacy-policy"
      />
      <Route
        component={[Function]}
        path="/terms-of-use"
      />
      <Route
        component={[Function]}
        path="/site-map"
      />
      <Route
        component={[Function]}
        path="/404"
      />
    </Switch>
    <Footer />
  </div>
</BrowserRouter>
`;
